templates/designate/bin/init.sh

```diff
@@ -19,8 +19,6 @@ set -ex
 # copies the result to the ephemeral /var/lib/config-data/merged volume.
 #
 # Secrets are obtained from ENV variables.
-export PASSWORD=${AdminPassword:?"Please specify a AdminPassword variable."}
-export TRANSPORTURL=${TransportURL:-""}
 export BACKENDURL=${BackendURL:-"redis://redis:6379/"}
```

Comment:
I guess you'll have to do the same thing for the BackendURL

Reply:
I don't think it is being used anywhere (or it gets overriden) because I don't see this specific value when I grep for those config options:
~/designate-operator > oc get secrets designate-config-data -o json | jq -r '.data["designate.conf"]' | base64 -d | grep backend_url
backend_url=memcached://127.0.0.1:11211
also, it doesn't appear on the controllers, so I will just remove those instances from the init.sh files.